packages/netlify/src/index.ts

```diff
@@ -52,6 +52,16 @@ export interface NetlifyIntegrationConfig {
 	 * @default disabled
 	 */
 	edgeMiddleware?: boolean;
+
+	/**
+	 * If enabled, Netlify Image CDN is used for image optimization.
+	 * This transforms images on-the-fly without impacting build times.
+	 * 
+	 * If disabled, Astro's built-in image optimization is run at build-time instead.
+	 * 
+	 * @default enabled
```

Comment:
```suggestion
	 * @default {true}
```

Reply:
We're also using enabled/disabled wording for the comment above. Personally, I prefer the english wording here as opposed to code syntax - but it's a preference thing. Is there a guideline on this in the Astro docs? cc @sarah11918

Reply:
This is JSDoc, which wants a **value**: https://jsdoc.app/tags-default

In this case, `enabled` isn't a valid value for this property

Reply:
Fair point - gonna change this here and in the other comment.